oak_functions/loader/src/tests.rs

```diff
@@ -430,3 +431,37 @@ fn test_format_bytes() {
     // Incorrect UTF-8 bytes, as per https://doc.rust-lang.org/std/string/struct.String.html#examples-3.
     assert_eq!("[0, 159, 146, 150]", format_bytes(&[0, 159, 146, 150]));
 }
+
+#[test]
+fn test_start_from_empty_endpoints() {
+    let check_empty = |endpoint: &mut EndPoint| {
```

Comment:
I don't think this needs to be a closure, it may be an inner fn (not 100% sure).

Reply:
Done! Thank you!